btcclient/config.go

```diff
@@ -32,6 +33,7 @@ type BTCConfig struct {
 	BlockCacheSize       uint64        `long:"block-cache-size" description:"Size of the Bitcoin blocks cache."`
 	MaxRetryTimes        uint          `long:"max-retry-times" description:"The max number of retries to an RPC call in case of failure."`
 	RetryInterval        time.Duration `long:"retry-interval" description:"The time interval between each retry."`
+	DisableTLS           bool          `long:"disable-tls" description:"Disable TLS for RPC connections"`
```

Comment:
why do we need this? how is it related to the issue we are fixing? please add the context in PR description

Reply:
yes, b/c in the original code `DisableTLS` is hard coded with `false`, it would cause errors when we didn't set up SSL for this GRPC server.